.github/issue_template.md

```diff
@@ -1,10 +1,10 @@
-# If your issue looks like hardware fault or a configuration problem this may be is the wrong place to post to.
-## Please consider using [RCGroups](https://rcgroups.com/forums/showthread.php?t=2464844), [Slack](https://slack.betaflight.tech) or any other forum before posting here or even better ask the manufacturer of the hardware you are using.
+# If your issue looks like a hardware fault or a configuration problem please don't raise an issue here.
+## Please consider using other user support options such as asking the manufacturer of the hardware you are using, [RCGroups](https://rcgroups.com/forums/showthread.php?t=2464844), [Slack](https://slack.betaflight.tech) or other user support forums & groups (e.g. facebook).
 
-## If you post here please follow these steps
-1.) Describe your problem
-2.) Include ways to reproduce the problem
-2.) Provide information about your flightcontroller and other components
-3.) Add the used configuration and firmware version
-4.) Create a diff and post it here in a code block (Simply put \`\`\` at the start and end of the diff block)
+## If you believe there is an issue with the firmware itself please follow these steps:
+1.) Describe your problem,
```

Comment:
hi there sorry about this but i am not new to quadcopters but i am to race drones, this my first day on here and trying to get a grip hold,  i think firmware is ok although the guy i bought it off reported the throttle glitch,
i have only flew it for 5 mins so  far and seemed ok except for it seems to need calibrating and i do not know how to perform it.
many thanks
Craig.